actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -458,11 +465,15 @@ private void orgCreationMetrics(Request actorMessage) {
           metricsResponseGenerator(responseFormat, periodStr, getViewData(orgId, orgName));
       sender().tell(response, self());
     } catch (ProjectCommonException e) {
-      ProjectLogger.log("Some error occurs", e);
+      ProjectLogger.log(
+          "OrganisationMetricsActor:orgCreationMetrics ProjectCommonException occured "
```

Comment:
From:
"OrganisationMetricsActor:orgCreationMetrics ProjectCommonException occured "

To:
"OrganisationMetricsActor:orgCreationMetrics: Exception in getting org creation data: "